microsoft-edge/hosting/webview2/Tooling/WebDriver.md

```diff
@@ -0,0 +1,132 @@
+---
+description: Automate and test the WebView2 Control using Microsoft Edge Driver
+title: Automating and Testing WebView2 with Microsoft Edge Driver
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 03/17/2020
+ms.topic: reference
+ms.prod: microsoft-edge
+ms.technology: webview
+keywords: IWebView2, IWebView2WebView, webview2, webview, edge, ICoreWebView2, ICoreWebView2Host, Selenium, Microsoft Edge Driver
+---
+
+# Automating and Testing WebView2 with Microsoft Edge Driver
+Because WebView2 utilizes the Chromium web platform, WebView2 developers can take advantage of standard web tooling for debugging and automation. One such tool is the W3C [WebDriver](https://www.w3.org/TR/webdriver2/) API, which can be used to create automated tests that simulate user interactions.
+
+Here’s how to get started:
+
+## Step 1: Download WebView2API Sample
+If you do not have an existing WebView2 project, download our [WebView2API Sample application](https://github.com/MicrosoftEdge/WebView2Samples/tree/master/WebView2APISample#webview2-api-sample), a comprehensive sample of the latest WebView2 SDK. Please double check that you have satisfied these [prerequisites](https://github.com/MicrosoftEdge/WebView2Samples/tree/master/WebView2APISample#prerequisites).
+
+Once you have cloned the repo, build the project in Visual Studio. It should look like the following:
+
+![alt text](..\images\WebDriver\sampleapp.PNG)
+
+## Step 2: Install Microsoft Edge Driver
+Follow the instructions to install [Microsoft Edge Driver](https://docs.microsoft.com/microsoft-edge/webdriver-chromium#download-microsoft-edge-driver) the browser-specific driver required by Selenium to automate and test WebView2.
+
+It is important to make sure that the version of Microsoft Edge Driver matches the version of Microsoft Edge that the application uses. For the WebView2API Sample to work, make sure that your version of Microsoft Edge is greater than or equal to the supported version of our latest SDK release found [in our Release Notes](https://docs.microsoft.com/microsoft-edge/hosting/webview2/releasenotes). To find out what version of Microsoft Edge you currently have, load `edge://settings/help` in the browser.
+
+## Step 3: Add Selenium to the WebView2API Sample
+At this point you should have Microsoft Edge installed, built a WebView2 project, and installed Microsoft Edge Driver. Now, let’s get started using Selenium. 
+
+**Note:** Selenium supports C#, Java, Python, Javascript, and Ruby. However, this guide will be in C#. 
```

Comment:
Please use this syntax for your note:

```markdown
> [!NOTE]
> Selenium supports C#, Java, Python, Javascript, and Ruby. However, this guide will be in C#. 
```
It may not appear correctly in VS Code/your editor's Markdown renderer but it will render the Note correctly in preview. Read more here: https://review.docs.microsoft.com/en-us/help/contribute/markdown-reference?branch=master#alerts-note-tip-important-caution-warning